src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -680,6 +682,18 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_ScanAssembliesDecideLightweightMarshaler">
+     <ItemGroup>
+      <AssembliesToScan Include="$(WasmAppDir)\managed\*.dll" /> <!-- ### REALLY? ### -->
```

Comment:
@radical what would be the right set of assemblies to scan, @(_WasmAssembliesInternal)?